torch/nn/functional.py

```diff
@@ -2571,16 +2571,16 @@ def binary_cross_entropy_with_logits(input, target, weight=None, size_average=No
     return torch.binary_cross_entropy_with_logits(input, target, weight, pos_weight, reduction_enum)
 
 
-def _smooth_l1_loss(input, target):
-    # type: (Tensor, Tensor) -> Tensor
+def _smooth_l1_loss(input, target, beta=1.0):
```

Comment:
it doesn't look like this is actually used?

Reply:
yes- wasn't sure originally if I could just delete it. I'll take it out now.